File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)

{
    //Prompt the user for positive integer between 1 and 8
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //Print out the height the user choose
    int d = 7;
    for (int i = 0; i < n; i++)
    {
        //Print out spaces to right align the pyramid
        for (int j = 0; j < d; j++)
        {
            printf(" ");
        }
        for (int j = -1; j < i; j++)

        {
            printf("#");
        }
        printf("\n");
        d = d - 1;
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

n = 999
while(n < 1 or n > 8):
    n = get_int("Height: ")

space = n - 1
for i in range(n):
    for j in range(i + 1):
        for k in range(space):
            print(" ", end="")
        print("#", end="")
    space -= 1
    print()
